<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>253</width>
    <height>311</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::West</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Menu</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QSplitter" name="splitter">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <widget class="QWidget" name="">
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QLabel" name="label_prenom">
                <property name="text">
                 <string>&amp;Prenom</string>
                </property>
                <property name="buddy">
                 <cstring>lineEdit_prenom</cstring>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_nom">
                <property name="text">
                 <string>&amp;Nom</string>
                </property>
                <property name="buddy">
                 <cstring>lineEdit_nom</cstring>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_nom_3">
                <property name="text">
                 <string>Ville</string>
                </property>
                <property name="buddy">
                 <cstring>lineEdit_nom</cstring>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_nom_2">
                <property name="text">
                 <string>Age</string>
                </property>
                <property name="buddy">
                 <cstring>lineEdit_nom</cstring>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_nom_4">
                <property name="text">
                 <string>zipcode</string>
                </property>
                <property name="buddy">
                 <cstring>lineEdit_nom</cstring>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="">
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QLineEdit" name="lineEdit_nom"/>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_prenom"/>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_prenom_2"/>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_prenom_3"/>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_prenom_4"/>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_ok">
              <property name="text">
               <string>PushButton</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
       <zorder>label_prenom</zorder>
       <zorder>horizontalSpacer</zorder>
       <zorder></zorder>
       <zorder>splitter</zorder>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
   <zorder>lineEdit_nom</zorder>
   <zorder>lineEdit_prenom</zorder>
   <zorder>label_nom</zorder>
   <zorder>label_prenom</zorder>
   <zorder>pushButton_ok</zorder>
   <zorder></zorder>
   <zorder>horizontalSpacer</zorder>
   <zorder>lineEdit_prenom_2</zorder>
   <zorder>label_nom_2</zorder>
   <zorder>label_nom_3</zorder>
   <zorder>lineEdit_prenom_3</zorder>
   <zorder>label_nom_4</zorder>
   <zorder>lineEdit_prenom_4</zorder>
   <zorder>horizontalSpacer</zorder>
   <zorder>tabWidget</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>253</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionCreate"/>
   </widget>
   <widget class="QMenu" name="menuKit">
    <property name="title">
     <string>Ki&amp;t</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuKit"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionCreate">
   <property name="text">
    <string>Create</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEdit_nom</tabstop>
  <tabstop>lineEdit_prenom</tabstop>
  <tabstop>lineEdit_prenom_2</tabstop>
  <tabstop>lineEdit_prenom_3</tabstop>
  <tabstop>lineEdit_prenom_4</tabstop>
  <tabstop>pushButton_ok</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
